app/views/pages/developer-guide/pages/creating-page.liquid

```diff
@@ -11,7 +11,7 @@ So that you can follow the steps in this tutorial, you have to have a layout tha
 
 * [Pages](/developer-guide/pages/pages)
 * [Creating a Layout](/developer-guide/pages/creating-layout)
-* [Deploy or Sync](/get-started/hello-world/hello-world#step-6-deploy-or-sync)
+* [Working with the Code and Files](/get-started/working-with-the-code-and-files/)
```

Comment:
This could be a link to https://documentation.platformos.com/get-started/working-with-the-code-and-files/#upload-your-code-to-an-instance

Reply:
Thank you, I updated it.